README.md

```diff
@@ -45,17 +45,33 @@ with more configuration options.
 From within the root of the repo run the following command to build an image of
 `renterd` tagged `renterd`.
 
-```bash
-docker build -t renterd .
+### Production
+
+```sh
+docker build -t renterd:latest -f ./docker/Dockerfile .
+```
+
+### Testnet
+
+```sh
+docker build -t renterd:latest-testnet -f ./docker/Dockerfile.testnet .
```

Comment:
The 'version' should be at the end. e.g.
`renterd-testnet:latest`

Reply:
Probably we want something like:

```
### Production

```sh
# build mainnet image
docker build -t renterd:latest -f ./docker/Dockerfile .

# build testnet image
docker build --build-arg BUILD_TAGS='netgo testnet' -t renterd-testnet:latest -f ./docker/Dockerfile .
```

Reply:
this is completed and I verified it works as expected